### PR TITLE
DDPB-2900: Update supporting documents content

### PR DESCRIPTION
**Pull request by acsauk-mojd**
## Purpose
Adds an inset hint advising deputy that they only need to provide supporting docs if requested to and some minor wording changes.

Fixes DDPB-2900.

## Checklist
- [x] I have performed a self-review of my own code
- [ ] I have updated documentation (Confluence/ADR/tech debt doc) where relevant
- [ ] I have added tests to prove my work
- [ ] The product team have approved these changes

### Frontend
- [ ] I have run an in-browser accessibility test (e.g. WAVE, Lighthouse)
- [ ] There are no deprecated CSS classes noted in the profiler
- [x] Translations are used and the profiler doesn't identify any missing

### CODE REVIEW

File: client/src/AppBundle/Resources/views/Report/Document/step1.html.twig [hunk 1]
```diff
@@ -18,16 +20,25 @@
 
     {{ form_start(form, {attr: {novalidate: 'novalidate' }}) }}
 
+    <h2 class="heading-medium">{{ (page ~ '.step1Heading') | trans }}</h2>
```
Comment (gregtyler) [inline client/src/AppBundle/Resources/views/Report/Document/step1.html.twig]:
```suggestion
    <h2 class="govuk-heading-m">{{ (page ~ '.step1Heading') | trans }}</h2>
```

File: client/src/AppBundle/Resources/views/Report/Document/step1.html.twig [hunk 2]
```diff
@@ -18,16 +20,25 @@
...
+
+    <div class="govuk-inset-text">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                {{ (page ~ '.selectHint1') | trans }}
```
Comment (gregtyler) [inline client/src/AppBundle/Resources/views/Report/Document/step1.html.twig]: Ol' fussport Tyler
```suggestion
                <p>{{ (page ~ '.selectHint1') | trans }}</p>
```

File: client/src/AppBundle/Resources/views/Report/Document/step1.html.twig [hunk 3]
```diff
@@ -18,16 +20,25 @@
...
+
     {% if step == 1 %}
         <div class="form-section push--bottom">
             <div class="form-group flush--bottom {% if not form.wishToProvideDocumentation.vars.valid %}form-group-error{% endif %}">
                 {{ form_checkbox_group(form.wishToProvideDocumentation, 'form.wishToProvideDocumentation', {
                     'useFormGroup': false,
                     'fieldSetClass' : 'inline',
-                    'legendClass' : 'form-label-bold',
-                    'legendText' : 'stepPage.form.wishToUploadDocumentation.label' | trans(transOptions, translationDomain),
+                    'legendClass' : 'form-label-bold'
```
Comment (gregtyler) [inline client/src/AppBundle/Resources/views/Report/Document/step1.html.twig]:
```suggestion
                    'legendClass' : 'govuk-fieldset__legend--m'
```

File: client/src/AppBundle/Resources/views/Report/Document/step2.html.twig [hunk 4]
```diff
@@ -24,9 +24,15 @@
...
 
-    <div class="text">
-        <p>{{ (page ~ '.selectHelp') | trans }}</p>
-        <p>{{ (page ~ '.selectHint') | trans }}</p>
+    <p>{{ (page ~ '.selectHelp') | trans }}</p>
+
+    <div class="govuk-inset-text">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <b>{{ (page ~ '.selectHint1') | trans }}</b>
```
Comment (gregtyler) [inline client/src/AppBundle/Resources/views/Report/Document/step2.html.twig]: Ol' fusspot strikes again
```suggestion
                <p><strong>{{ (page ~ '.selectHint1') | trans }}</strong></p>
```

Comment (acsauk-mojd) [reply client/src/AppBundle/Resources/views/Report/Document/step2.html.twig]: You can see my extent of touching anything frontend has been VERY LIMITED of late 😂

File: client/src/AppBundle/Resources/views/Report/Document/step1.html.twig (hunk 2 above)
Comment (acsauk-mojd) [reply client/src/AppBundle/Resources/views/Report/Document/step1.html.twig]: Sooo this was actually a conscious design decision to match the prototype as it looked like there was no spacing between hint1 and hint2:

Screenshot 2020-03-26 at 11 05 44

Is there a more semantic way to lay this out?

Comment (gregtyler) [reply client/src/AppBundle/Resources/views/Report/Document/step1.html.twig]: I see, yep, probably just this then:
```
             <p>
                 <strong>{{ (page ~ '.selectHint1') | trans }}</strong>
                 <br>
                 {{ (page ~ '.selectHint2') | trans }}
             </p>
```

Comment (acsauk-mojd) [reply client/src/AppBundle/Resources/views/Report/Document/step1.html.twig]: Makes sense 👍 to confuse matters, I provided the prototype for step2 as I couldn't see a prototype for step1. Instructions on the ticket was to add the same inset message but with the first line as regular text. I'll add the `<br>` to both.

File: client/src/AppBundle/Resources/views/Report/Document/step1.html.twig [hunk 5]
```diff
@@ -18,16 +20,26 @@
...
     {{ form_start(form, {attr: {novalidate: 'novalidate' }}) }}
 
+    <h2 class="govuk-heading-m">{{ (page ~ '.step1Heading') | trans }}</h2>
+
+    <div class="govuk-inset-text">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                {{ (page ~ '.selectHint1') | trans }}
+                <br>
+                <p>{{ (page ~ '.selectHint2') | trans }}</p>
```
Comment (gregtyler) [inline client/src/AppBundle/Resources/views/Report/Document/step1.html.twig]: `<p>` is unnecessary now (`<br>` provides a newline, `<div>` provides the block-level element)
```suggestion
                {{ (page ~ '.selectHint2') | trans }}
```

File: client/src/AppBundle/Resources/views/Report/Document/step2.html.twig [hunk 6]
```diff
@@ -24,9 +24,16 @@
...
-        <p>{{ (page ~ '.selectHelp') | trans }}</p>
-        <p>{{ (page ~ '.selectHint') | trans }}</p>
+    <p>{{ (page ~ '.selectHelp') | trans }}</p>
+
+    <div class="govuk-inset-text">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <strong>{{ (page ~ '.selectHint1') | trans }}</strong>
+                <br>
+                <p>{{ (page ~ '.selectHint2') | trans }}</p>
```
Comment (gregtyler) [inline client/src/AppBundle/Resources/views/Report/Document/step2.html.twig]: Y lo mismo:
```suggestion
                {{ (page ~ '.selectHint2') | trans }}
```

Comment (acsauk-mojd) [reply client/src/AppBundle/Resources/views/Report/Document/step2.html.twig]: Está bien, no hay problema

**Pull request merged**